src/org/usfirst/frc2175/command/group/RunIntakeInGroup.java

```diff
@@ -12,13 +12,15 @@
  */
 public class RunIntakeInGroup extends CommandGroup {
 
+    int speed = sideBeltSpeedForward.getSideBeltSpeedForward();
+
     public RunIntakeInGroup(DriverStation driverStation,
             RobotSubsystems robotSubsystems) {
 
         addParallel(new RunDreamIntakeSideBeltAtSpeedCommand(driverStation,
-                robotSubsystems, 1));
+                robotSubsystems, speed));
         addSequential(new RunDreamIntakeMainBeltAtSpeedCommand(driverStation,
-                robotSubsystems, 1));
+                robotSubsystems, speed));
```

Comment:
These shouldn't both be using the same speed value, since we can set different speeds for both belts in the properties file.